test/DynamoCoreTests/CoreDynTests.cs

```diff
@@ -63,6 +63,29 @@ public void AddSubtractMapReduceFilterBasic()
             AssertPreviewValue("41279a88-2f0b-4bd3-bef1-1be693df5c7e", new[] { 6, 7, 8, 9, 10 });
         }
 
+        /// <summary>
+        /// Confirm that node model derived nodes deserialize into correct node state.
+        /// </summary>
+        [Test]
+        public void verifyNodeStates()
```

Comment:
can you also add a json version of this test?

Reply:
@mjkkirschner should cover xml and json now